commands/commands.js

```diff
@@ -35,37 +32,62 @@ export const askName = () => {
     return name;
 }
 
+const userDetailsArray = [];
 const storeData = () => {
-    const userDetails = JSON.stringify(finalData);
-    const readerFile = fs.existsSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8");
+    userDetailsArray.length = 0
```

Comment:
Why is this line required?? as empty array's length is always 0

Reply:
The array length is set to zero because everytime a user opts to hear more than 2 jokes, the jokes are doubled in the json file. It was done to erase the existing files in the array after each entry and push the existing file through content and new data through finalData